target/linux/ramips/dts/mt7620a_netgear_ex6130.dts

```diff
@@ -0,0 +1,57 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6130", "ralink,mt7620a-soc";
+	model = "Netgear EX6130";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_g {
+			label = "ex6130:green:power";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		power_a {
+			label = "ex6130:amber:power";
+			gpios = <&gpio2 28 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
If you have two power leds, wouldn't it make sense to exploit both for aliases?

Reply:
I think of something like this:
https://github.com/openwrt/openwrt/blob/master/target/linux/lantiq/files/arch/mips/boot/dts/FRITZ7412.dts#L17-L21